packages/docs/docs/communications/organizing-communications.md

```diff
@@ -0,0 +1,480 @@
+# Organizing Communications
+
+# Introduction
+
+Communications in a healthcare context (modeled as [Communication](/docs/api/fhir/resources/communication) resources) can include many scenarios – patient to physician, physician to physician, device to physician, and more. This results in a large volume of communications within this context, so ensuring that they are well organized is vital. For example, a provider may have staff on call to respond to patient messages. Without proper organization, messages may not reach the correct staff member.
+
+This guide covers different ways that you can organize communications and how to think about different properties on the `Communication` element. It will cover:
+
+- Common communication organization patterns, including `Communication.category`, `Communication.topic`, and threading.
```

Comment:
I think these should be two seperate sections. 
1. How to organize `Communication` resources into threads, to model things like email or SMS threads. Mention `topic` here
2. How to "tag" the threads / individual messages with the `category` field